ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-ref.yaml

```diff
@@ -1,5 +1,6 @@
 ref:
   as: ipi-deprovision-deprovision
+  best_effort: true
```

Comment:
note: this only takes effect if a job that runs this step is configured with `allow_best_effort_post_steps: true`